packages/ui/src/ui/pages/users/UsersPageEditor/UsersPageEditor.tsx

```diff
@@ -328,6 +345,20 @@ class UsersPageEditor extends React.Component<Props, State> {
                                 ...errors,
                             ],
                         },
+                        {
+                            type: 'tab-vertical',
+                            name: 'password',
+                            title: 'Change Password',
```

Comment:
The title looks inaccurate, lets replace it with:
```ts
title: 'Password'
```

<img width="948" alt="image" src="https://github.com/user-attachments/assets/96dbf399-0d80-4cd5-bb9b-2abc2fe47cdc">